### PR TITLE
Adding PHP Simple devfile tests.

### PR DESCRIPTION
**Pull request by ScrewTSW**
### What does this PR do?
Adds a PHP Simple devfile test.
PR is dependant on #17583 

### What issues does this PR fix or reference?
#17462 

#### Release Notes
N/A

#### Docs PR
N/A

### CODE REVIEW

File: tests/e2e/tests/devfiles/PHPSimple.spec.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+const buildTaskLinkExpectedText: string = 'A process is now listening on port 8080.';
+const stack: string = 'PHP Simple';
+
+suite(`${stack} test`, async () => {
+    suite (`Create ${stack} workspace`, async () => {
+        workspaceHandling.createAndOpenWorkspace(stack);
+        projectAndFileTests.waitWorkspaceReadinessNoSubfolder(workspaceSampleName);
+    });
+
+    suite('Language server validation', async () => {
```
Comment (Katka92) [inline tests/e2e/tests/devfiles/PHPSimple.spec.ts]: I would suggest to follow the same structure of the test that is in other tests - open the file first, then execute commands (so LS has enough time to start) and then execute LS tests.

File: tests/e2e/tests/devfiles/PHPSimple.spec.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        projectAndFileTests.openFile(fileFolderPath, tabTitle);
+        prepareEditorForLSTests();
+        commonLsTests.errorHighlighting(tabTitle, `error_text;`, 14);
+        commonLsTests.suggestionInvoking(tabTitle, 14, 5, 'phpinfo');
+        commonLsTests.autocomplete(tabTitle, 14, 5, 'phpinfo');
```
Comment (Katka92) [inline tests/e2e/tests/devfiles/PHPSimple.spec.ts]: I don't see the value in running those tests that way - they are testing the same... The purpose was to test the autocomplete of variable name and suggestion of method.

**Review by Katka92 — CHANGES_REQUESTED** [review]

File: tests/e2e/tests/devfiles/PHPSimple.spec.ts (hunk 2 above)
Comment (ScrewTSW) [reply tests/e2e/tests/devfiles/PHPSimple.spec.ts]: Oh, I see now. Will try to rewrite the logic.

File: tests/e2e/tests/devfiles/PHPSimple.spec.ts (hunk 1 above)
Comment (ScrewTSW) [reply tests/e2e/tests/devfiles/PHPSimple.spec.ts]: LS is initialized practically instantly, but I'm all for keeping the tests consistent. thank you for the notice.

File: tests/e2e/tests/devfiles/PHPSimple.spec.ts (hunk 2 above)
Comment (ScrewTSW) [reply tests/e2e/tests/devfiles/PHPSimple.spec.ts]: fixed.

File: tests/e2e/tests/devfiles/PHPSimple.spec.ts (hunk 1 above)
Comment (ScrewTSW) [reply tests/e2e/tests/devfiles/PHPSimple.spec.ts]: fixed.

**Review by Katka92 — APPROVED** [review]

**Pull request merged**